config.js

```diff
@@ -51,5 +51,10 @@ module.exports = {
       'shooting-clubs': 'https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=shootclub/',
       museums: 'https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=Museums/'
     }
+  },
+  pdf: {
+    url: env !== 'production' ?
+    `http://${process.env.PDF_CONVERTER_HOST}:${process.env.PDF_CONVERTER_PORT}/convert` :
+    process.env.PDF_CONVERTER_URL
```

Comment:
I'm not sure I grasp the reasoning behind using a different set of environment variables for production and "not production".